cluster-autoscaler/cloudprovider/azure/azure_template_test.go

```diff
@@ -20,10 +20,11 @@ import (
 	"fmt"
 	"testing"
 
-	//nolint SA1019 - deprecated package
-
+	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2022-08-01/compute"
```

Comment:
nit: this line is missing ` //nolint SA1019 - deprecated package` if we compare with the fork